release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3023,6 +3023,70 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.18.20
+[id="ocp-4-18-20_{context}"]
+=== RHSA-2025:10767 - {product-title} {product-version}.20 bug fix and security update
+
+Issued: 17 July 2025
+
+{product-title} release {product-version}.20 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:10767[RHSA-2025:10767] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:10768[RHSA-2025:10768] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.20 --pullspecs
+----
+
+[id="ocp-4-18-20-bug-fixes_{context}"]
+==== Bug fixes
+
+* Before this update, a subset of endpoints on the console backend were gated by `TokenReview` requests to the API server. In some cases, the API server would throttle these requests, causing slower load times in the UI. With this release, the `TokenReview` gating was removed from all but one of our endpoints resulting in improved performance. (link:https://issues.redhat.com/browse/OCPBUGS-58317[OCPBUGS-58317])
+
+* Before this update, useful error messages were not generated when the`oc adm node-image create` command failed. With this release, the `oc adm node-image create` command generates error messages when the command fails. (link:https://issues.redhat.com/browse/OCPBUGS-58233[OCPBUGS-58233])
+
+* Before this update, the HAProxy configuration used the `/version` endpoint for health checks causing unreliable health checks to be generated. With this release, the liveness probe is customized to use `/livez?exclude=etcd&exclude=log` on {ibm-cloud-title} for more accurate health checks avoiding disruptions due to inappropriate probe configurations on Hypershift, while retaining `/version` for other platforms. (link:https://issues.redhat.com/browse/OCPBUGS-58126[OCPBUGS-58126])
```

Comment:
```suggestion
* Before this update, the HAProxy configuration used the `/version` endpoint for health checks causing unreliable health checks to be generated. With this release, the liveness probe is customized to use `/livez?exclude=etcd&exclude=log` on {ibm-cloud-title} for more accurate health checks. This change avoids disruptions due to inappropriate probe configurations on hosted control planes, while retaining `/version` for other platforms. (link:https://issues.redhat.com/browse/OCPBUGS-58126[OCPBUGS-58126])
```
"hypershift" is the upstream name, so this is a merge blocker; please update (see https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit?gid=1375785039#gid=1375785039)